.ci/cico_updates_minikube.sh

```diff
@@ -88,7 +81,7 @@ waitCheUpdateInstall() {
 
 testUpdates() {
   "${OPERATOR_REPO}"/olm/testUpdate.sh ${PLATFORM} ${CHANNEL} ${NAMESPACE}
-  printInfo "Successfully installed Eclipse Che previous version."
+  echo "Successfully installed Eclipse Che previous version."
```

Comment:
It could be a bit better:

echo "[INFO] Successfully installed Eclipse Che previous version."

Reply:
+1

Reply:
Also I'll use function <functionNAME> nomenclature in this file.